### ISSUE TITLE
[HW2] bootstrap에 관해 질문있습니다.

### ISSUE THREAD

**Issue by jangsus1** [swsnu/swppfall2019#88]
Bootstrap을 이용해 디자인을 일부 바꿔서 일반 input과 button 태그가 아니라 Form.Group,Form.Control과 Button 클래스를 사용하게 되었습니다. 그래서 spec에서 요구되는 태그 id 들을 클래스의 id 로 설정해주기만 했는데 혹시 채점 과정에 영향이 있을까요??

**Comment by ktaebum** [comment]
괜찮습니다. 그런 component도 src를 보면 결국 해당 jsx의 wrapper class여서 영향은 없습니다

**Comment by kyunggeun-lee** [comment]
No, **you should not change the html tag names.** If you do, some test cases in our grading script will fail, and you won't get the score for those test cases even though you have implemented everything else correctly.
